### PR TITLE
Registering subdomain for verification with improvmx

### PR DESCRIPTION
**Pull request by MysticalPvE**
Register 76048217._improvmx.dhilipan.is-a.dev with TXT record pointing to 76048217dc3e4aa88034771330290cd0.

### CODE REVIEW

**Comment by MysticalPvE** [comment]
Improvmx login, used for verification (for email forwarding).

File: domains/76048217._improvmx.dhilipan.json
Comment (Stef-00012) [inline domains/76048217._improvmx.dhilipan.json:1]: subdomains prefixed with `_' are not alllowed (here you have `_improvmx`)

there are only a few exceptions like github pages or discord verification but those don't include improvmx

also i've never seen this kind of verification with improvmx

**Review by Stef-DP — DISMISSED** [review]

**Comment by MysticalPvE** [comment]
Screenshot_2024-08-13-17-21-50-890_com android chrome

I asked orher maintainers from discord and they said it would get accepted if for verification purposes. I'll even delete it after verification. (Note: if your domain is already claimed by someone, a verification for improvmx using DNS entry has to be done)

**Review by iostpa — APPROVED** [review]
Merging this since user mistyped their email while setting up ImprovMX with this domain.

**Pull request merged**